shared/editor/rules/embeds.ts

```diff
@@ -18,7 +19,10 @@ function isLinkClose(token: Token) {
   return token.type === "link_close";
 }
 
-export default function linksToEmbeds(embeds: EmbedDescriptor[]) {
+export default function linksToEmbeds(
```

Comment:
You know, I can't find any usage of this method – remove?

Reply:
`server/editor/renderToHtml.ts` and `shared/editor/nodes/Embed.tsx`?

Reply:
Well... I guess my code search was broken.